src/components/clusterWizard/ClusterWizardStep.css

```diff
@@ -0,0 +1,7 @@
+.cluster-wizard-step .pf-c-wizard__nav-item {
+  white-space: nowrap;
```

Comment:
This will lead to longer step names not getting wrapped and the warning icon potentially not being visible. How about just spacing the icon from the text with `&nbsp;` - that will prevent from breaking the line between the text and icon while allowing the text to break in case it is long.

Reply:
Exactly. But we do not have such names now and we should prevent using them in the future.